base/gfspapp/manage_server.go

```diff
@@ -331,3 +331,15 @@ func (g *GfSpBaseApp) GfSpNotifyMigrateSwapOut(ctx context.Context, req *gfspser
 	}
 	return &gfspserver.GfSpNotifyMigrateSwapOutResponse{}, nil
 }
+
+func (g *GfSpBaseApp) GfSpQuerySPByOperatorAddress(ctx context.Context, req *gfspserver.GfSpQuerySPByOperatorAddressRequest) (
```

Comment:
Add UTs.

Reply:
It is super weried that `QuerySPByOperatorAddress` is an interface of a `Manager`...

Reply:
Fixed. Currently, sp cache pool is added to the gateway.